dockerfiles/README.md

```diff
@@ -3,43 +3,35 @@ Docker images
 
 Docker images related to HPCC are structured as follows
 
-hpccsystems/platform-build-base
+**hpccsystems/platform-build-base**
 
 This image contains all the development packages required to build the hpcc platform,
-but no HPCC code or sources. It changes rarely. The current version is tagged 7.8 and
-is based on Ubuntu 18.04 base image
+but no HPCC code or sources. It changes rarely. The current version is tagged 7.10 and
+is based on Ubuntu 20.04 base image
 
 **hpccsystems/platform-build**
 
-Building this image builds an installation package (.deb file) for a specified git tag
-of the HPCC platform sources. The Dockerfile takes two arguments, naming the version of
-the platform-build-base image to use, and the git tag to use. Sources are fetched from
+Building this image build and installs the HPCC codebase for a specified git tag
```

Comment:
"Building this image build" not quite sure what it means should it be "builds.."?

Reply:
Yes. Changed